devhelp/src/Makefile.am

```diff
@@ -25,6 +25,7 @@ devhelp_la_CFLAGS = \
 
 devhelp_la_LIBADD = \
 	$(DEVHELP_LIBS) \
+	$(COMMONLIBS) \
```

Comment:
I now fixed this part in 2d1dee0d9a135f2d6d29c4eead6c6e13e5d10dcd (and the markdown one in 856b3d9a0d065866c8117b031fe82087543b3f6f)